core-plugins/docs/SSHAction-action.md

```diff
@@ -0,0 +1,56 @@
+# SSH Action
+
+
+Description
+-----------
+Establishes an SSH conncetion with remote machine to execute command on that machine.
+
+
+Use Case
+--------
+This action can be used when data needs to be copied to HDFS from a remote server before the pipleline starts.
+It can also be used to move data for archiving before the pipeline starts.
+
+
+Properties
+----------
+
+**host:** The host name of the remote machine where the command needs to be executed.
+
+**port:** The port used to make SSH connection. Defaults to 22.
+
+**user:** The username used to connect to host.
+
+**privateKeyFile:** The file path to Private key.
+
+**password:** The password associated with private key
+
+**cmd:** The command to be executed on the remote host.
```

Comment:
I think we ultimately are going to use secure storage, so this is only temporary.